design/high-level-reactivity.md

```diff
@@ -94,6 +100,31 @@ Before, we gave an example of a GraphQL response for a simple query. If you make
 }
 ```
 
+// I think following example is simple and complete.
+
+```
+{
+  me: {
+    username: "sashko",
+    lists: [
+      {
+        id: 1,
+        name: "My first todo list",
+      }
+    ],
+
+    __deps: {
+      _self: { key: '12341234', version: 3 },
+      lists: { key: '35232345', version: 4 }
+    }
+  }
+}
+```
+
+// Thinking about _self and lists. Is that because to avoid data re-fetching of lists when the username changes?
+// If so, what are the types we want to do it?
+// Is that something we can do it in the GraphQL Schema level.
```

Comment:
The `_deps` are specified at the schema level. The client also needs to request them, but that's easy because it has the schema from the introspection. Relay already does this to add the `id` field to your request even if you don't ask for it.